### PR TITLE
Adding png files, updates to booktitle and new copyright.md file

### PR DESCRIPTION
**Pull request by sirnish**
- Adding png files, updates to booktitle (correcting release edition) and separated copyright info to copyright.md file.
- Also .md files are updated to use Image's local reference instead of github link.

### CODE REVIEW

File: booktitle.md [hunk 1]
```diff
@@ -1,38 +1,35 @@
 <div align="center">
 
-<!--![Image](https://github.com/sirnish/FreeRTOS-Kernel-Book-toc/blob/main/media/freeRTOS.png)-->
+<!--![Image](https://github.com/FreeRTOS/FreeRTOS-Kernel-Book/blob/main/media/freeRTOS.png)-->
```
Comment (n9wxu) [inline booktitle.md:3]: Must we use URL's or can we use local file references?  If I am viewing this offline it will not work.

**Review by n9wxu — COMMENTED** [review]
questioning the image references.

File: booktitle.md (hunk 1 above)
Comment (sirnish) [reply booktitle.md:3]: Thanks for pointing it out. It makes more sense to use local reference for offline usage. Accepted and updated.

File: booktitle.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
```
Comment (aggarg) [inline booktitle.md:3]: I guess we can remove these comments?

**Review by aggarg — APPROVED** [review]

File: booktitle.md (hunk 2 above)
Comment (sirnish) [reply booktitle.md]: Thanks... Removed.

**Review by aggarg — APPROVED** [review]

**Pull request merged**